ts/src/ascendex.ts

```diff
@@ -416,7 +521,9 @@ export default class ascendex extends Exchange {
         //         ]
         //     }
         //
-        const cash = await this.v1PublicGetCashAssets (params);
```

Comment:
I've removed cash-assets as it was undocumented endpoint and didnt either add any important value (it just includedsomewhat obsolete  decomissioned type of cash-tokens with -FZ/ -FT suffixes)